types/react-native-autocomplete-input/index.d.ts

```diff
@@ -61,9 +55,9 @@ export interface AutocompleteProps<T> extends TextInputProps {
 
     /**
      * function
-     * onShowResult will be called when the autocomplete suggestions appear or disappear.
+     * onShowResults will be called when the autocomplete suggestions appear or disappear.
      */
-    onShowResult?(showResults: boolean): void;
+    onShowResults?(showResults: boolean): void;
```

Comment:
Fixed typo in the name of this prop.

See https://github.com/mrlaessig/react-native-autocomplete-input#props and https://github.com/mrlaessig/react-native-autocomplete-input/blob/a011865598dd0b37c371892366f4212c18095735/index.js#L32